.github/workflows/bdd-integration-tests.yml

```diff
@@ -1,4 +1,4 @@
-name: acapy-integration-tests
+name: BDD Integration Tests
```

Comment:
Might want to scope test execution to `paths` that contain code and/or test definitions, so that tests are not executed when no code changes are submitted. Same for other workflows.

Reply:
Ya. That is a good idea. I could add that to this PR.